data/json/items/magazine/762x25.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "ppsh71drum",
```

Comment:
Items need to go in alphabetical ordering of `id` to reduce future merge conflicts